include/libtorrent/sha1_hash.hpp

```diff
@@ -35,10 +35,8 @@ POSSIBILITY OF SUCH DAMAGE.
 
 #include <cctype>
 #include <algorithm>
-#include <string>
 #include <cstring>
 
-#include "libtorrent/config.hpp"
```

Comment:
this is required for ``TORRENT_USE_IOSTREAM`` and ``TORRENT_EXTRA_EXPORT`` and probably a bunch of other macros